demos/python_demos/colorization_demo/colorization_demo.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+"""
+ Copyright (c) 2018 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from openvino.inference_engine import IENetwork, IECore
+import cv2 as cv
+import numpy as np
+import os
+from argparse import ArgumentParser, SUPPRESS
+import xml.etree.ElementTree as ET
+import re
+import logging as log
+import sys
+
+
+def build_arg():
+    parser = ArgumentParser(add_help=False)
+    in_args = parser.add_argument_group('Options')
+    in_args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Help with the script.')
+    in_args.add_argument("-m", "--model", help="Required. Path to .xml file with pre-trained model",
+                         required=True, type=str)
+    in_args.add_argument("-d", "--device",
+                         help="Optional. Specify target device for infer: CPU, GPU, FPGA, HDDL or MYRIAD. "
+                         "Default: CPU",
+                         default="CPU", type=str)
+    in_args.add_argument("-i", "--input", help="Optional. Path to a test video file.", type=str)
+    in_args.add_argument("-n", "--no_show", help="Optional. Disable display of results on screen.",
+                         action='store_true', default=False)
+    in_args.add_argument("-v", "--verbose", help="Optional. Enable display of processing logs on screen.",
+                         action='store_true', default=False)
+    return parser
+
+
+def get_parameters(xml_path):
+    root = ET.parse(xml_path).getroot()
+    pattern = r'data\[([0-9]*\.*[0-9]*)\]'
+    mean_data = root.findall("./meta_data/cli_parameters/mean_values")[0]
+    mean_const = float(re.match(pattern, mean_data.attrib['value']).group(1))
+    return mean_const
+
+
+def get_model_files(config_xml_in):
+    model_path = os.path.splitext(config_xml_in)[0]
+    weights_bin_in = model_path + ".bin"
+    level_up = os.path.split(os.path.split(model_path)[0])[0]
+    coeffs_in = os.path.join(level_up, os.path.split(level_up)[1]) + ".npy"
+    return config_xml_in, weights_bin_in, coeffs_in
+
+
+if __name__ == '__main__':
+    args = build_arg().parse_args()
+    config_xml, weights_bin, coeffs = get_model_files(args.model)
+
+    # mean are stored in the source caffe model and passed to IR
```

Comment:
```suggestion
    # mean is stored in the source caffe model and passed to IR
```
or `means are` if there are multiple numbers

Reply:
@Wovchena done)